src/interfaces/IParameterReader.h

```diff
@@ -0,0 +1,19 @@
+/// \file      IParameterReader.h
+/// \brief
+/// \date      Jan 05, 2022
+/// \author    My Linh Wuerzburger
+/// \copyright <2015-2021> Forschungszentrum Juelich. All rights reserved
+
+#ifndef ARTSS_INTERFACES_IPARAMETERREADER_H
+#define ARTSS_INTERFACES_IPARAMETERREADER_H
+
+#include <string>
+#include "../utility/settings/Settings.h"
+using return_parameter_reader = std::tuple<bool, Settings::data_assimilation::field_changes>;
```

Comment:
why an extra `bool` if `field_changes` already contains the member `changed`?

Reply:
`changed` is for the field changes only and the extra bool is for the parameter changes